tests/phpunit/tests/fonts/font-library/wpFontFamily/install.php

```diff
@@ -0,0 +1,562 @@
+			$font_file = path_join( static::$fonts_dir, $font_file );
+			$this->assertFileDoesNotExist( $font_file, "Font file [{$font_file}] should not exist in the fonts/ directory after installing" );
+		}
+		$font = new WP_Font_Family( $font_data );
+
+		// Test.
+		$font->install();
+		foreach ( $expected as $font_file ) {
+			$font_file = path_join( static::$fonts_dir, $font_file );
+			$this->assertFileExists( $font_file, "Font file [{$font_file}] should exists in the fonts/ directory after installing" );
+		}
+		$this->assertInstanceOf( WP_Post::class, $font->get_font_post(), 'Font post should exist after install' );
+	}
+
+	/**
+	 * Data provider.
+	 *
+	 * @return array[]
+	 */
+	public function data_should_download_fontfaces() {
+		return array(
+			'1 font face to download'  => array(
+				'font_data' => array(
+					'name'       => 'Piazzolla',
+					'slug'       => 'piazzolla',
+					'fontFamily' => 'Piazzolla',
+					'fontFace'   => array(
+						array(
+							'fontFamily'      => 'Piazzolla',
+							'fontStyle'       => 'italic',
+							'fontWeight'      => '400',
+							'src'             => 'http://fonts.gstatic.com/s/piazzolla/v33/N0b72SlTPu5rIkWIZjVgI-TckS03oGpPETyEJ88Rbvi0_TzOzKcQhZqx3gX9BRy5m5M.ttf',
+							'downloadFromUrl' => 'http://fonts.gstatic.com/s/piazzolla/v33/N0b72SlTPu5rIkWIZjVgI-TckS03oGpPETyEJ88Rbvi0_TzOzKcQhZqx3gX9BRy5m5M.ttf',
+						),
+					),
+				),
+				'expected'  => array( 'piazzolla_italic_400.ttf' ),
+			),
+			'2 font faces to download' => array(
+				'font_data' => array(
+					'name'       => 'Lato',
+					'slug'       => 'lato',
+					'fontFamily' => 'Lato',
+					'fontFace'   => array(
+						array(
+							'fontFamily'      => 'Lato',
+							'fontStyle'       => 'normal',
+							'fontWeight'      => '400',
+							'src'             => 'http://fonts.gstatic.com/s/lato/v24/S6uyw4BMUTPHvxk6XweuBCY.ttf',
+							'downloadFromUrl' => 'http://fonts.gstatic.com/s/lato/v24/S6uyw4BMUTPHvxk6XweuBCY.ttf',
+						),
+						array(
+							'fontFamily'      => 'Lato',
+							'fontStyle'       => 'italic',
+							'fontWeight'      => '400',
+							'src'             => 'http://fonts.gstatic.com/s/lato/v24/S6u8w4BMUTPHjxswWyWrFCbw7A.ttf',
+							'downloadFromUrl' => 'http://fonts.gstatic.com/s/lato/v24/S6u8w4BMUTPHjxswWyWrFCbw7A.ttf',
+						),
+					),
+				),
+				'expected'  => array( 'lato_normal_400.ttf', 'lato_italic_400.ttf' ),
+			),
+		);
+	}
+
+	/**
+	 * @dataProvider data_should_move_local_fontfaces
+	 *
+	 * @param array $font_data  Font family data in theme.json format.
+	 * @param array $files_data Files data in $_FILES format.
+	 * @param array $expected   Expected font filename(s).
+	 */
+	public function test_should_move_local_fontfaces( $font_data, array $files_data, array $expected ) {
+		// Set up the temporary files.
+		foreach ( $files_data as $file ) {
+			if ( 'font/ttf' === $file['type'] ) {
+				copy( __DIR__ . '/../../../data/fonts/Merriweather.ttf', $file['tmp_name'] );
```

Comment:
TEST_DIR const?

Reply:
Thanks for flagging that. Done here: https://github.com/WordPress/wordpress-develop/commit/e218ec8b94e679da12c013bf7ce617f169ee0738

Reply:
Confirmed that https://github.com/WordPress/wordpress-develop/pull/5285/commits/e218ec8b94e679da12c013bf7ce617f169ee0738 did indeed resolve this. I did a follow-up commit https://github.com/WordPress/wordpress-develop/pull/5285/commits/d6534cc08a4262f02c858d59a7e1a1d4c945b0b5 to remove the `path_join()` to simplify and be consistent in Core tests ✅ 

Making this as resolved, which will fold / hide it. I don't think leaving it unresolved helps to provide additional context.